__tests__/app.test.js

```diff
@@ -3,34 +3,93 @@ const request = require("supertest");
 const connection = require("../db/connection");
 const seed = require("../db/seeds/seed");
 const data = require("../db/data/test-data");
+const endpoints = require("../endpoints.json")
 
 afterAll(() => {
-    return connection.end()
+  return connection.end();
 });
 
 beforeEach(() => {
-    return seed(data)
-})
+  return seed(data);
+});
 
-describe("GET api/topics", () => {
-    describe("Connecting to path", () => {
-        test("Should return status 200", () => {
-            return request(app).get("/api/topics").expect(200)
+describe("GET /api/topics", () => {
+  describe("Connecting to path", () => {
+    test("Should return status 200", () => {
+      return request(app).get("/api/topics").expect(200);
+    });
+    test("Should return the data from the path", () => {
+      return request(app)
+        .get("/api/topics")
+        .expect(200)
```

Comment:
Because you are checking `expect(200)` in the test before this one, you can remove this line from the current test